templates/search/blocks/search_filters.html

```diff
@@ -11,7 +11,7 @@ <h2 class="tna-heading-l search-sort-view__heading">
         {# render hidden inputs for fields controlled by other forms #}
         {% render_fields_as_hidden form exclude='sort_by' %}
 
-        <input type="submit" value="Update" class="search-sort-view__form-submit">
+        <input type="submit" value="Sort by" class="search-sort-view__form-submit">
```

Comment:
```suggestion
        <input type="submit" value="Sort" class="search-sort-view__form-submit">
```
Hi @JohnHeeryTNA just a query/suggestion if you'd like to consider - feels abrupt ending with **by** in a button

Reply:
Hmmn, that's what was suggested from the user testing, we can alway update it if there are better suggestions..